CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/) 
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.8.2]
+
+### Modified
+* the dems.md and rtc_product_guide.md to remove the legacy DEM option in the HyP3 processing for RTC
```

Comment:
```suggestion
* the RTC and DEM guides to inform users that the legacy DEM options are no longer available
```

Reply:
DEM is not guide. How about " the DEM description and RTC product guide to inform users that the legacy DEM options are no longer available"

Reply:
That works too.